.rubocop_fixme.yml

```diff
@@ -20,6 +21,7 @@ Metrics/AbcSize:
     - 'lib/oregon_digital/triple_powered_properties/work_behavior.rb'
     - 'lib/tasks/oregon_digital/csv_ingest.rake'
     - 'spec/support/helpers/session_helpers.rb'
+    - 'lib/oregon_digital/controlled_vocabularies/extended_location.rb'
```

Comment:
I think this should be addressed in the code instead of overridden and hidden

Reply:
I just put it here so I could work on getting tests ironed out. I didnt want a need to refactor to block me from getting tests passing. Ill make a note next time.

Reply:
I'm curious if you're able to run these tests locally instead of testing by way of CircleCI? It should be a lot faster to run them locally..but I might not understand if there's a blocker preventing you from doing that?

Reply:
More like a not fully fleshed out development habit. From what I understand, to run the tests you have to bring docker down, spin it up in testing mode, run the tests, then to fire the app up, you have to switch back over right?